core/src/main/scala/org/apache/spark/sql/catalyst/analyzer/TiAuthorizationRule.scala

```diff
@@ -49,17 +51,15 @@ case class TiAuthorizationRule(getOrCreateTiContext: SparkSession => TiContext)(
           .foreach(TiAuthorization.authorizeForSetDatabase(_, tiAuthorization))
       }
       sd
-    case dt @ DescribeRelation(
```

Comment:
will describe relations be supported?

Reply:
it will be supported. i delete it because the auth for describe statement has been done in TiCatalog.loadTable.